File: 58. Length of Last Word/solution.js
var lengthOfLastWord = function(s) 
{
    let word = ""

    for (let i = s.length - 1; i >= 0; --i)
    {
        if (word.length == 0 && s[i] == ' ')
            continue
        else
        {
            if (s[i] == ' ')
                return word.length

            word += s[i]
        }
    }    

    return word.length
};

File: 58. Length of Last Word/solution.cpp
class Solution 
{
    public:
        int lengthOfLastWord(string s) 
        {
            string word = "";

            for (int i = size(s) - 1; i >= 0; --i)
            {
                if (word.empty() && s[i] == ' ')
                    continue;
                else
                {
                    if (s[i] == ' ')
                        return word.size();
                    
                    word.push_back(s[i]);
                }
            }

            return word.size();
        }
};